resources/icons/20px/object-view.svg

```diff
@@ -0,0 +1,16 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_4201_351)">
+<path d="M5 0H1C0.447715 0 0 0.447715 0 1V5H2L2 2H5V0Z" fill="#5F6B7C"/>
+<path d="M18 15V18H15V20H19C19.5523 20 20 19.5523 20 19V15H18Z" fill="#5F6B7C"/>
+<path d="M0 15H2L2 18H5V20H1C0.447715 20 0 19.5523 0 19V15Z" fill="#5F6B7C"/>
+<path d="M15 0H19C19.5523 0 20 0.447715 20 1V5H18V2H15V0Z" fill="#5F6B7C"/>
+<path d="M9.55078 2.12511L3.74133 5.25192L10 9L16.2586 5.25192L10.5 2.12511C10.1926 1.9583 9.85821 1.9583 9.55078 2.12511Z" fill="#5F6B7C"/>
+<path d="M9.5 10.0001L3.10575 5.99622C3.03733 6.126 3 6.27107 3 6.42122V12.6996C3 13.0404 3.19229 13.355 3.50386 13.5241L9.39275 16.9497C9.42767 16.9687 9.46348 16.9855 9.5 17.0001V10.0001Z" fill="#5F6B7C"/>
+<path d="M16.8942 5.99615C16.9626 6.12594 16.9999 6.27101 16.9999 6.42116V12.6996C16.9999 13.0403 16.8076 13.355 16.4961 13.524L10.6073 16.9497C10.5724 16.9686 10.5365 16.9854 10.5 17.0001L10.4999 10.0001L16.8942 5.99615Z" fill="#5F6B7C"/>
+</g>
+<defs>
+<clipPath id="clip0_4201_351">
+<rect width="20" height="20" fill="white"/>
+</clipPath>
+</defs>
```

Comment:
Similar to [above](https://github.com/palantir/blueprint/pull/7113/files#r1878349395). Can we also remove the unnecessary `clipPath`?
```svg
<svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
<path d="M5 0H1C0.447715 0 0 0.447715 0 1V5H2L2 2H5V0Z"/>
<path d="M18 15V18H15V20H19C19.5523 20 20 19.5523 20 19V15H18Z"/>
<path d="M0 15H2L2 18H5V20H1C0.447715 20 0 19.5523 0 19V15Z"/>
<path d="M15 0H19C19.5523 0 20 0.447715 20 1V5H18V2H15V0Z"/>
<path d="M9.55078 2.12511L3.74133 5.25192L10 9L16.2586 5.25192L10.5 2.12511C10.1926 1.9583 9.85821 1.9583 9.55078 2.12511Z"/>
<path d="M9.5 10.0001L3.10575 5.99622C3.03733 6.126 3 6.27107 3 6.42122V12.6996C3 13.0404 3.19229 13.355 3.50386 13.5241L9.39275 16.9497C9.42767 16.9687 9.46348 16.9855 9.5 17.0001V10.0001Z"/>
<path d="M16.8942 5.99615C16.9626 6.12594 16.9999 6.27101 16.9999 6.42116V12.6996C16.9999 13.0403 16.8076 13.355 16.4961 13.524L10.6073 16.9497C10.5724 16.9686 10.5365 16.9854 10.5 17.0001L10.4999 10.0001L16.8942 5.99615Z"/>
</svg>
```